grin/src/Pipeline/Optimizations.hs

```diff
@@ -39,7 +39,7 @@ import Transformations.Optimising.SimpleDeadParameterElimination (simpleDeadPara
 import Transformations.Optimising.CSE (commonSubExpressionElimination)
 import Transformations.Optimising.CaseCopyPropagation (caseCopyPropagation)
 import Transformations.Optimising.GeneralizedUnboxing (generalizedUnboxing)
-import Transformations.Optimising.ArityRaisingSimple (arityRaising)
+import Transformations.Optimising.ArityRaising (arityRaising)
```

Comment:
Please revert this change. The new arity raising implementation should be supported. It is well tested via the idris frontend. The test was deliberately was turned off as it is too specific for the implementation of the old arity raising implementation.

Reply:
Done